packages/neo4j/src/account.ts

```diff
@@ -0,0 +1,96 @@
+import neo4j from "neo4j-driver"
+import type { Account } from "next-auth"
+import { v4 as uuid } from "uuid"
+
+export const linkAccount = async (
+  neo4jSession: typeof neo4j.Session,
+  data: Account
+) => {
+  const { userId, providerAccountId, provider, ...accountData } = data
+
+  // TODO: Question, account.expires is an int. here
```

Comment:
As far as I know, there is no `expires` on `Account`. You might be thinking `expires_at`, which comes from here: https://github.com/panva/node-openid-client/blob/772306cc82b72ace9361f4a945249a5c35a8f921/types/index.d.ts#L714-L718

Since we just forward these values, we will respect their type as well, so this is kept as a number.